docs/api/ref/schemas/product_nutrition_v3.yaml

```diff
@@ -0,0 +1,814 @@
+                    minimum: 0
+                    description: |
+                      The nutrition data on the package can be per serving, per 100g or per 100ml.
+                      When the data is given per serving, the actual quantity that defines one serving may vary 
+                      between products and should be stored in this field.
+  
+                      This is essential to understand to which quantity values in `nutrients` apply for.
+
+                      For example, if the label states "per 250g", then this field should be *250*.
+                    example: 250
+                  per_unit:
+                    type: string
+                    description: |
+                      The nutrition data on the package can be per serving, per 100g or per 100ml.
+                      When the data is given per serving, the actual unit that defines one serving may vary 
+                      between products and should be stored in this field.
+  
+                      This is essential to understand to which quantity values in `nutrients` apply for.
+
+                      For example, if the label states "per 250g", then this field should be *g*.
+                    example: "g"
+                  source:
+                    type: string
+                    enum:
+                      - packaging
+                      - manufacturer
+                      - usda
+                      - estimate
+                      - preferred
+                    description: |
+                      The nutrition data of products can be obtained through several sources.
+
+                      The *preferred* source takes values from multiple sources (by priority: manufacturer, packaging, usda, estimate) 
+                      and normalizes all values to the same unit: g for weights, kJ for energy and energy-kj, kcal for energy-kcal.
+
+                      The *preferred* source is the one used to display nutrition data of products.
+                  source_description:
+                    type: string
+                    description: |
+                      A description of the source used for this nutrition set.
+
+                      This provides more information on how and when the source was used.
+                    examples:
+                      - USDA non-branded foods 2025/04
+                      - Import from org-nestle-france through Equadis
+                      - "Estimate from ingredients”, “Estimate from category: Olive oils"
+                  last_updated_t:
+                    type: integer
+                    description: |
+                      A timestamp indicating when this nutrition set was last updated.
+                    examples:
+                      - 1631270265
+                      - 1512153487
+                  nutrients:
+                    type: object
+                    description: |
+                      All known nutrients for the product.
```

Comment:
We should indicate that new nutrients are regularly added, and that clients should not break if they encounter an unexpected nutrient.